tests/SwaggerUiTest.php

```diff
@@ -54,10 +54,13 @@ private function createContainer(): ContainerInterface
             CacheInterface::class => new ArrayCache(),
             DataResponseFactoryInterface::class => DataResponseFactory::class,
             ResponseFactoryInterface::class => Psr17Factory::class,
-            ViewRenderer::class => function (Container $container) {
+            ViewRenderer::class => function (
```

Comment:
```suggestion
            ViewRenderer::class => static function (
```

Reply:
This function shouldn't be static, otherwise, it will fail, because `$this` is used